### DISCUSSION TITLE
What musician named all of his 🎸🎸 Lucille?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#926] [Galaxy Trivia] [answered]
**Bonus:** What kind of music did the musician play? Name the best-known song.

**Comment by peterbaccaro** [comment, marked as answer]
The musician who named all of his guitars "Lucille" is the legendary blues guitarist **B.B. King**. B.B. King was known for his soulful playing style and his partnership with his beloved Gibson ES-355 guitar, which he affectionately called "Lucille." According to the story, the name "Lucille" originated from an incident that occurred early in B.B. King's career. He was playing at a venue in Arkansas when a fight broke out among the audience, and the building caught fire. B.B. King escaped, but he realized that he had left his guitar inside. Risking his life, he ran back into the burning building to retrieve his beloved instrument. Later, he found out that the fight had been over a woman named Lucille. From that point on, B.B. King decided to name all of his guitars "Lucille" as a reminder to avoid such reckless behaviour and to cherish the things that are important to him.

B.B. King was primarily known as a **blues musician**. He was one of the most influential and celebrated blues guitarists of all time. His playing style was characterized by expressive, soulful guitar solos and his distinctive vibrato technique.

B.B. King's best-known song is arguably "**The Thrill is Gone**". Released in 1969, it became a major hit and one of his signature songs. "The Thrill is Gone" showcased B.B. King's masterful guitar work and heartfelt vocals, capturing the essence of his emotional and introspective approach to the blues. The song is notable for its melancholic tone and memorable guitar solos, which have made it a classic in the blues genre.

Throughout his career, B.B. King recorded numerous other notable songs, including "Every Day I Have the Blues," "Sweet Little Angel," "Rock Me Baby," and "Lucille," among many others.
